### PR TITLE
feat(replays): Add a tooltip to show relative time on event chapters

### LINKED ISSUE #34225: Add a button to toggle between relative and absolute timestamps in the chapters component
The existing breadcrumb component has a button at the top of the "time" column to toggle between the two modes:
Screen Shot 2022-05-05 at 12 31 56 AM

Since we are showing a time in the Chapter component, it could work in the same way.

**NOTE:**
When we render the relative time, it should be relative to the start of the replay: `event.startTimestamp`.

The time increments from there. You can imagine an example page where the data looks like this:

| ------ | ---- | --- |
| event type | abs. time | rel. time |
| ------ | --- | --- |
| navigate | 3:45:00pm | 0s |
| click | 3:46:30 | 90s |
| click | 3:46:45 | 105s |
| fetch | 3:46:46 | 106s |

The relative time is a positive number, incrementing after the replay start time

### PR DESCRIPTION
**Pull request by SDaian**
### Description

- Add a tooltip on relativeTime to show absolute time on event chapters components

Closes #34225 

### Screenshots

image

----

### CODE REVIEW

File: static/app/components/replays/playerRelativeTime.tsx [hunk 1]
```diff
@@ -12,9 +14,38 @@ const PlayerRelativeTime = ({relativeTime, timestamp}: Props) => {
     return <div />;
   }
 
-  return <Value>{showPlayerTime(timestamp, relativeTime)}</Value>;
+  return (
+    <Wrapper>
+      <Tooltip
+        title={getDateFormatted(timestamp)}
```
Comment (billyvg) [inline static/app/components/replays/playerRelativeTime.tsx:20]: Can we use the `<DateTime>` component here?

Comment (SDaian) [reply static/app/components/replays/playerRelativeTime.tsx]: Changed to `<DateTime />` component

File: static/app/components/replays/playerRelativeTime.tsx [hunk 2]
```diff
@@ -12,9 +15,38 @@ const PlayerRelativeTime = ({relativeTime, timestamp}: Props) => {
...
+  :before {
+    content: '';
+    display: block;
+    width: 1px;
+    top: 0;
+    bottom: 0;
+    left: 50%;
+    transform: translate(-50%);
+    position: absolute;
+  }
```
Comment (billyvg) [inline static/app/components/replays/playerRelativeTime.tsx:47]: Is this needed at all?

File: static/app/components/replays/playerRelativeTime.tsx [hunk 3]
```diff
@@ -12,9 +15,38 @@ const PlayerRelativeTime = ({relativeTime, timestamp}: Props) => {
...
+`;
+
```
Comment (billyvg) [inline static/app/components/replays/playerRelativeTime.tsx:49]: Actually, is the Wrapper needed at all? The tooltip should position itself just fine.

Comment (SDaian) [reply static/app/components/replays/playerRelativeTime.tsx:49]: Yep, you were right, don't know why but seems to be in other places that are using a `<Wrapper>` for the Tooltip component, just realize doesn't need it anymore. Deleted.

**Review by billyvg — APPROVED** [review]

**Pull request merged**